metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/query/request/SearchRequestHandler.java

```diff
@@ -488,7 +490,7 @@ private void addMissingAggregationValueToAggregationMetadata(@Nonnull final Stri
             || originalMetadata.getFilterValues().stream().noneMatch(entry -> entry.getValue().equals(value))
     ) {
       // No aggregation found for filtered value -- inject one!
-      originalMetadata.getAggregations().put(value, 0L);
+      originalMetadata.getAggregations().put(value, MAX_VALUE_FOR_FILTER_VALUE);
```

Comment:
What is the implication of this...

Reply:
This basically makes it so that if the user tries to filter on something that does not exist, we report it correctly to them on the product side

Reply:
This means that filters in the query will have priority over aggregations added just because they appear in search results